dev-infrastructure/templates/global-acr.bicep

```diff
@@ -23,16 +27,55 @@ resource globalMSI 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31'
   name: globalMSIName
 }
 
+//
+//   O C P   A C R
+//
+
 module ocpAcr '../modules/acr/acr.bicep' = {
   name: ocpAcrName
   params: {
     acrName: ocpAcrName
     acrSku: ocpAcrSku
+
     location: location
     zoneRedundant: determineZoneRedundancyForRegion(location, ocpAcrZoneRedundantMode)
   }
 }
 
+module ocpCaching '../modules/acr/cache.bicep' = {
+  name: '${ocpAcrName}-caching'
+  params: {
+    acrName: ocpAcrName
+    location: location
+    quayRepositoriesToCache: [
+      {
+        ruleName: 'openshiftReleaseDev'
+        sourceRepo: 'quay.io/openshift-release-dev/*'
+        targetRepo: 'openshift-release-dev/*'
+        userIdentifier: 'quay-username'
+        passwordIdentifier: 'quay-password'
+        loginServer: 'quay.io'
+      }
+    ]
+    purgeJobs: [
```

Comment:
We probably want to swap `purgeJobs` -> retention policy since these are untagged manifests now.  

https://learn.microsoft.com/en-us/azure/container-registry/container-registry-retention-policy

Reply:
ack. i will change